src/services/auth/utils.test.ts

```diff
@@ -0,0 +1,106 @@
+import crypto from 'crypto';
+import jwt from 'jsonwebtoken';
+import { v4 } from 'uuid';
+
+import { FastifyRequest } from 'fastify';
+
+import build, { clearDatabase } from '../../../test/app';
+import { AUTH_TOKEN_JWT_SECRET } from '../../utils/config';
+import { InvalidSession, MemberNotFound, OrphanSession } from '../../utils/errors';
+import { Member } from '../member/entities/member';
+import { BOB, saveMember } from '../member/test/fixtures/members';
+import { fetchMemberInSession, verifyMemberInAuthToken, verifyMemberInSession } from './utils';
+
+const buildRequest = (memberId?: string) =>
+  ({
+    session: {
+      get: () => memberId,
+      delete: jest.fn(),
+    },
+    log: { warn: jest.fn() },
+  } as unknown as FastifyRequest & { member?: Member });
```

Comment:
Not sure about the types here. Is it for testing purposes only?

Reply:
Yes, test only. It's not easy to mock a fastify request.